### DISCUSSION TITLE
Aflæsning kan ikke ske ved lave kwh priser

### DISCUSSION THREAD

**Discussion by haej** [rndfm/elspotpris#142] [Ideas]
![image](https://user-images.githubusercontent.com/35633838/219963006-c7a335c7-17d9-4726-8458-8e23b23dca22.png)

**Comment by rndfm** [comment]
Jeg har lavet en løsning så prisen står over baren hvis baren er for lille til at indeholde prisen.

![image](https://user-images.githubusercontent.com/46886958/220395755-65775e03-f544-4fd0-8132-c2d7663fee51.png)

**Comment by haej** [comment]
Tak. Sendt fra min iPhone +45 5239 4076Med venlig hilsenHans Erik JacobsenDen 21. feb. 2023 kl. 16.58 skrev Christian Reinholdt ***@***.***>:﻿
Jeg har lavet en løsning så prisen står over baren hvis baren er for lille til at indeholde prisen.
